packages/zoe/src/contractSupport/durability.js

```diff
@@ -53,34 +49,80 @@ harden(provideEmptySeat);
 /**
  * For making singletons, so that each baggage carries a separate kind definition (albeit of the definer)
  *
+ * @template {(baggage: import('@agoric/ertp').Baggage, ...rest: any[]) => any} M Maker function
  * @param {import('@agoric/vat-data').Baggage} baggage
  * @param {string} category diagnostic tag
+ * @param {M} prepareSingleton
+ * @param {{}} categoryPowers
  */
-export const provideChildBaggage = (baggage, category) => {
-  const baggageSet = provideDurableSetStore(baggage, `${category}Set`);
-  return Far('childBaggageManager', {
-    // TODO(types) infer args
+export const provideCategorySingletons = (
```

Comment:
This seems unrelated to the rest of the PR. `provideCategorySingletons` and `makeSingletons` are defined, but not used here. Have I missed something?

Reply:
Remnant of an interim state. Forgot to drop them